ignite/templates/ibc/oracle.go

```diff
@@ -253,14 +268,34 @@ func clientCliQueryOracleModify(replacer placeholder.Replacer, opts *OracleOptio
 // Deprecated: This function is no longer maintained.
 func clientCliTxOracleModify(replacer placeholder.Replacer, opts *OracleOptions) genny.RunFn {
 	return func(r *genny.Runner) error {
-		path := filepath.Join(opts.AppPath, "x", opts.ModuleName, "client/cli/tx.go")
+		path := filepath.Join(opts.AppPath, "x", opts.ModuleName, "module/autocli.go")
```

Comment:
The oracle commands were hidden and are not working now. We don't need to maintain it anymore. We should discuss what to do with this later